### ISSUE TITLE
collapse lib with subtree

### ISSUE THREAD

**Issue by cazlo** [cazlo/Auto_Jobs_Applier_AIHawk#2]
there is an open dependency in the main repo to a lib repo. 

the lib repo is only used by the main project.

refactor this to 1 project to simplify dependency mgmt and remove potentially malicious open dependency.

**Comment by cazlo** [comment]
depends on #1 

(github has no issue dependency feature yet :\ https://github.com/orgs/community/discussions/4928 )

**Comment by cazlo** [comment]
lib repo https://github.com/feder-cr/lib_resume_builder_AIHawk/tree/main

**Comment by cazlo** [comment]
closed won't do. see #1 for reasoning
